index.js

```diff
@@ -31,16 +32,18 @@ module.exports = function (archive, target, opts, cb) {
   var entries = {}
   var watcher
 
-  if (opts.live) {
+  if (opts.watch && archive.live) {
     watcher = chokidar.watch([target], {
       persistent: true,
       ignored: opts.ignore
     })
     watcher.once('ready', function () {
       watcher.on('add', function (file, stat) {
+        status.emit('file watched', {path: file, mode: 'created'})
```

Comment:
The `file imported` event doesn't come until after the file is done importing. There can be a lot of lag between a user action (adding file) and the import being done.

Tried to handle this lag without adding an event (as I had done in the old live file watching on CLI) and couldn't find any good solution.

With this event we can show the user what is happening, for example in the CLI I'm thinking about showing:

```
Importing 5 files...

Added: my-file.csv
Updated: another-thing.csv
```

The count will show files that have been triggered by a `file watched` event and the bottom show files after importing is done (`file imported`).